build_manifest.json

```diff
@@ -2,102 +2,142 @@
   "barretenberg-x86_64-linux-clang": {
     "buildDir": "circuits/cpp/barretenberg/cpp",
     "dockerfile": "dockerfiles/Dockerfile.x86_64-linux-clang",
-    "rebuildPatterns": ["^circuits/cpp/barretenberg/cpp/"],
+    "rebuildPatterns": [
+      "^circuits/cpp/barretenberg/cpp/"
+    ],
```

Comment:
Incorrect formatting has been annoying me for a while so I've created [an issue](https://github.com/AztecProtocol/aztec-packages/issues/1566) for this.